modules/View.js

```diff
@@ -56,6 +56,72 @@
       if (!options.noActivate) {
         this.activate();
       }
+      // this.updateDelegateEvents();
+    },
+
+
+    /**
+    * called by updateDelegateEvents to wrap callback functions of event calls
+    * with event triggers
+    * @param method {function} callback method of event
+    * @param eventName {String} description of event type 
+    * @return modified mehtod with before/after event triggers
+    * @method trackEvents
+    */
+    trackEvents : function(method, eventName){
+      var self = this;
+      var methodCopy = method;
+      method = _.bind(function(){
+        var uuid = (new Date()).getTime().toString(16)+Math.floor(1E7*Math.random()).toString(16);
+        this.trigger('clickEventTiming', {
+          uuid: uuid,
+          type: 'clickEvent',
+          state: 'start',
+          eventName: eventName,
+          time: Date.now(),
+        });
+
+        methodCopy.call(self);
+
+        this.trigger('routetiming', {
```

Comment:
same here: 'post-dom-event-callback' or something like that. Trigger with the same uuid (like you've done) as the pre-dom-event-callback.  Also add the listener in the constructor.